wp-content/plugins/core/src/Templates/Content/Panels/Hero.php

```diff
@@ -21,28 +23,97 @@ public function get_data(): array {
 
 	public function get_mapped_panel_data(): array {
 		$data = [
-			'text_color'    => $this->text_color(),
-			'layout'        => $this->get_layout(),
-			'image'         => $this->get_image(),
-			'content_block' => $this->get_content_block(),
+			'row_classes'     => $this->get_row_classes(),
+			'content_classes' => $this->get_content_classes(),
+			'image'           => $this->get_image(),
+			'content_block'   => $this->get_content_block(),
 		];
 
 		return $data;
 	}
 
+	protected function is_split_layout(): bool {
+		if ( $this->panel_vars[ HeroPanel::FIELD_LAYOUT ] === HeroPanel::FIELD_LAYOUT_OPTION_CONTENT_SPLIT_LEFT || $this->panel_vars[ HeroPanel::FIELD_LAYOUT ] === HeroPanel::FIELD_LAYOUT_OPTION_CONTENT_SPLIT_RIGHT ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	protected function get_classes(): string {
```

Comment:
To give ourselves some ready to go CSS "hooks" out of the box, we are outputting a class on the parent wrapper for each option. If you can make sure we are doing that here as well, that'd be great!

Reply:
@ryanurban Do you have an example of another panel that we're doing this on?  Just want to make sure I'm keeping things consistent.

Reply:
Media + Image & Interstitial

Reply:
@ryanurban Can you clarify what you’re referring to here?  I’m currently adding a layout class to the wrapper, along with a utility BG color class.  Do you also want the text color class on here as well?  Seems like we don’t really need it.